recipe/reader.go

```diff
@@ -99,7 +99,7 @@ func (r *Reader) readDir(path string) (recipes []Recipe, err error) {
 	for _, entry := range entries {
 		recipe, err := r.readFile(filepath.Join(path, entry.Name()))
 		if err != nil {
-			continue
```

Comment:
@StewartJingga the cause of this issue seems to be this `continue` statement when reading files from a directory. What happens is while reading a list of recipes we were skipping the recipes which had an issue or error in them, and continued running the rest.

But when there is just one file in the directory, and that too is invalid, it just reports the error as recipe not found.

Reply:
Nice, catch!

Also, we can try wrapping the most the errors there using `fmt.Errorf` to enable debugging easier.

Reply:
btw we still need the continue syntax if we don't want it to break on invalid recipe.

If you want to break, we can just return error instead of logging it

Reply:
We are not breaking it even now, instead of `continue` we are logging the error, and it continues reading other recipes in the directory.

Reply:
CMIIW if we are not continuing, it will continue to next code, which will put empty recipe on recipe list.

Reply:
Ah Yes, makes sense.